XML package casuing windows build to crash

It seems that something has happend aroundt h 5/12/2022 that has caused the build to start failing.

The error given by GitHub is:

>  ── R CMD build ─────────────────────────────────────────────────────────────────
> pdflatex not found! Not building PDF manual.
> * checking for file 'D:\a\BrailleR\BrailleR/DESCRIPTION' ... OK
> * preparing 'BrailleR':
> * checking DESCRIPTION meta-information ... OK
> * installing the package to build vignettes
>       -----------------------------------
> ERROR: dependency 'XML' is not available for package 'BrailleR'
> * removing 'C:/Users/RUNNER~1/AppData/Local/Temp/Rtmp8aeTKo/Rinst[17](https://github.com/ajrgodfrey/BrailleR/actions/runs/3626274394/jobs/6115112295#step:9:18)245f8c7[20](https://github.com/ajrgodfrey/BrailleR/actions/runs/3626274394/jobs/6115112295#step:9:21)0/BrailleR'
>       -----------------------------------
> ERROR: package installation failed
> Error: Error in proc$get_built_file() : Build process failed
> Calls: <Anonymous> ... build_package -> with_envvar -> force -> <Anonymous>
> Execution halted
> Error: Process completed with exit code 1.

The first point of call will be to try and reduce the imports and see if it still fails.

--- a/NAMESPACE
+++ b/NAMESPACE
@@ -22,7 +22,7 @@ importFrom("stats", "aggregate", "anova", "as.ts", "coef", "cor", "lm", "median"
 importFrom("tools", "Rd2ex")
 importFrom("utils", "browseURL", "create.post", "download.file", "file.edit", "head", "menu", "packageVersion", "read.csv", "read.table", "savehistory", "Sweave", "str", "tail", "unzip", "write.csv") 
 importFrom("whisker", "whisker.render")
-import(XML)
+importFrom("XML", "addAttributes", "newXMLDoc", "newXMLNode", "ensureNamespace", "newXMLTextNode", "xmlName", "xmlRoot", "saveXML", "getNodeSet", "xmlGetAttr", "addChildren", "removeChildren", "xmlChildren")
 importFrom("xtable", "xtable")
 S3method(AddXML,default)
   S3method(AddXML,boxplot)

